apps/nextjs-website/src/messages/it.json

```diff
@@ -105,5 +105,27 @@
             "subscribedLabel": "Iscrizione confermata",
             "unsubscribeLabel": "Annulla iscrizione"
         }
+    },
+    "errors": {
```

Comment:
I would maintain consistency across all the sentences; some conclude with a period, while others do not